### PR TITLE
Add account username blacklist

### PR DESCRIPTION
**Pull request by tdorianh**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -25,7 +25,6 @@
 
 def init():
     configLog = logging.getLogger("config")
-
```
Comment (the-maldridge) [inline main.py]: There should not be a whitespace change in this commit.

File: validate.py [hunk 2]
```diff
@@ -17,9 +18,11 @@ def user(self, user):
         uValid = re.compile(ur'^[A-Za-z0-9.-_]{1,30}$')
         if re.search(uValid, user):
             self.logger.debug("User %s passed regex check", user)
+            for word in self.blacklist:
+                if (word in user.lower()):
```
Comment (the-maldridge) [inline validate.py]: this isn't a good check, what if I wanted the username "IAmGroot"?  Because it contains 'root' it would fail your blacklist.

**Comment by the-maldridge** [comment]
You may also wish to check for non-ascii characters.  I'm not sure what ZFS will do, but LDAP will happily accept characters for which there aren't glyphs and it will be just fine accepting strings that can't be typed with a US keyboard.  You could do that with an ordinal range check, but its probably way easier to just check it with a regex.

**Comment by kulinacs** [comment]
Looks good to me. I'll test deployment tomorrow and merge if successful.

**Pull request merged**